torch/nn/modules/crop.py

```diff
@@ -0,0 +1,70 @@
+import torch
+
+from .module import Module
+from .. import functional as F
+
+
+class _CropBase(Module):
+
+    def __init__(self, size, mode):
+        super(Module, self).__init__()
+        self.size = size
+        self.mode = mode
+        if self.mode != 'center':
+            raise NotImplementedError("CropNd only supports mode center crop")
+
+    def _offset(self, initial, target):
+        crop = torch.FloatTensor([initial]).sub(target).div(-2)
```

Comment:
Do you need to create a new `FloatTensor` for this? You can probably use the `math` library for this, and maybe the integer division operator `//` to avoid the `ceil`/`floor`.